.github/workflows/ci.yml

```diff
@@ -17,28 +17,27 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2
+    - name: Install uv
+      uses: astral-sh/setup-uv@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        enable-cache: true
+        # Install a specific version of uv.
+        version: "0.4.x"
+    - name: Set up Python ${{ matrix.python-version }}
+      run: uv python install ${{ matrix.python-version }}
     - name: Install apt dependencies
       run: |
         sudo apt-get update && sudo apt-get -y install plantuml libxml2-dev libxslt-dev python3-dev
-    - name: Install pip & test dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install --upgrade setuptools
-        pip install pytest pre-commit
     - name: Install dependencies for Python
       run: |
-        pip install -c requirements.txt -e .[tests,sphinx,svgsupport,aafiguresupport,mathsupport,rawhtmlsupport]
+        uv sync --all-extras --python ${{ matrix.python-version }}
     - name: Lint
       run: |
-        pre-commit run --all-files --show-diff-on-failure
-      if: matrix.python-version == '3.8'
```

Comment:
This was the bugfix mentioned in the PR description. Python 3.8 isn't present in the testing version matrix, so it was never executed (see previous CI runs on `main`).